### PR TITLE
Add MysqlDump command

### PR DESCRIPTION
**Pull request by felipanico**
#51

### CODE REVIEW

File: app/Commands/MySqlDump.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle(Process $process)
+    {
+        return $process->dockerCompose(
+            sprintf('exec -e MYSQL_PWD=%s mysql mysqldump -u root', env('DB_PASSWORD')),
```
Comment (gabrielboliveira) [inline app/Commands/MySqlDump.php]: What about the env user? 

```suggestion
            sprintf(
                'exec -e MYSQL_PWD=%s mysql mysqldump -u %s',
                env('DB_PASSWORD'),
                env('DB_USERNAME')
            ),
```

**Comment by fabriciojs** [comment]
@felipanico would be nice to make this on top of the branch `dbpolito:1.0` that we have approved already - just missing the merge. That way you will already use the new and better way of writing the commands. Please take a look and let me know what you think.

**Comment by felipanico** [comment]
@fabriciojs wich of these branches?

image

**Comment by fabriciojs** [comment]
@felipanico the branch `dbpolito:1.0` - that is in Polito`s fork.

**Comment by fabriciojs** [comment]
@felipanico thanks for this contribution! This is a very important piece.

Please can you rebase this with `1.0-rc` and the new Command Builder structure?

**Comment by felipanico** [comment]
@fabriciojs ok!

**Comment by fabriciojs** [comment]
@felipanico @dbpolito @gabrielboliveira - would be great to have this. I don't wanna close it although we might think about new ways of doing it, in a more generic fashion as well.

**Pull request closed without merging**